### PR TITLE
[PREC-141] Improve graph look

### PR DESCRIPTION
**Pull request by braemJef**
- strokeColor now colors different things:
  - Edge lines
  - Icon color
  - Icon circle stroke
- color now colors only the icon circle background

Screenshot 2022-10-27 at 09 32 59
Screenshot 2022-10-27 at 09 33 25

### CODE REVIEW

File: src/layers/graph-layer/graph-layer.js [hunk 1]
```diff
@@ -274,45 +276,98 @@ export default class GraphLayer extends Layer {
...
+        data: edges,
         updateTriggers,
-        extensions,
+        parameters: {
+          // circles will be flat on the map when the altitude column is not used
+          depthTest: this.config.columns.altitude?.fieldIdx > -1
+        },
 
-        // LineLayer stuff
+        // LineLayer properties
```
Comment (kkostov) [inline src/layers/graph-layer/graph-layer.js:314]: Much better 😍

**Review by kkostov — APPROVED** [review]
Nice! ⛴️

File: src/layers/graph-layer/graph-layer.js (hunk 1 above)
Comment (taeymma) [reply src/layers/graph-layer/graph-layer.js:314]: haha, noticed that as well

**Review by taeymma — APPROVED** [review]
🚀

**Pull request merged**